public/resources/remix/index.js

```diff
@@ -6,122 +6,136 @@
+    head.appendChild(analyticsHtml);
   }
 
-  function setupBar(err, $) {
+  function setupBar(err) {
     if(err) {
       console.log("[Thimble Error] Unable to inject Remix UI. Error was: `" + err + "`");
       return;
     }
 
     var isTouchDevice = 'ontouchstart' in document.documentElement;
-    var detailsBar = $(".details-bar");
-    detailsBar.attr("style", "");
+    var detailsBar = document.querySelector(".details-bar");
+    detailsBar.setAttribute("style", "");
 
     if(isTouchDevice) {
-      detailsBar.addClass("touch-mode");
-    } else {
-      detailsBar.addClass("mouse-mode");
+      detailsBar.classList.add("touch-mode");
+    }
+    else {
+      detailsBar.classList.add("mouse-mode");
     }
 
-    detailsBar.on("click", ".thimble-button",function(){
-      detailsBar.removeClass("collapsed");
-      return false;
+    detailsBar.addEventListener("click", function(event) {
+      if (event.target.className == "thimble-button") {
+        detailsBar.classList.remove("collapsed");
+        return false;
+      }
     });
 
-    detailsBar.on("click", ".close-details-bar", function(){
-      detailsBar.addClass("collapsed");
-      return false;
+    detailsBar.addEventListener("click", function(event) {
+      if (event.target.className == "close-details-bar") {
+        detailsBar.classList.add("collapsed");
+        return false;
+      }
     });
 
-    $(".mouse-mode").on("mouseenter",function(){
-      detailsBar.removeClass("collapsed");
+    document.querySelector(".mouse-mode").addEventListener("mouseenter", function(){
+      detailsBar.classList.remove("collapsed");
     });
 
-    $(".mouse-mode").on("mouseleave",function(){
-      detailsBar.addClass("collapsed");
+    document.querySelector(".mouse-mode").addEventListener("mouseleave", function(){
+      detailsBar.classList.add("collapsed");
     });
   }
 
-  function injectDetailsBar($, metadata, callback) {
-    $.ajax({
-      url: metadata.host + "/projects/remix-bar",
-      cache: false,
-      accepts: "text/html",
-      dataType: "html",
-      data: {
-        host: metadata.host,
-        id: metadata.projectId,
-        title: metadata.projectTitle,
-        author: metadata.projectAuthor,
-        updated: metadata.dateUpdated
+  function injectDetailsBar(metadata, callback) {
+    var xmlhttp = new XMLHttpRequest();
+    xmlhttp.responseType = "text";
+    var url = metadata.host + "/projects/remix-bar";
+    var data = {
+      host: metadata.host,
+      id: metadata.projectId,
+      title: metadata.projectTitle,
+      author: metadata.projectAuthor,
+      updated: metadata.dateUpdated
+    };
+
+    xmlhttp.onreadystatechange = function() {
+      if (xmlhttp.readyState == XMLHttpRequest.DONE ) {
```

Comment:
you don't use `==` in JavaScript, unless you are *absolutely certain* you need a converting value. As such, always test with `===` unless you can explain and justify why JS should be allowed to try to automatically convert values for the purposes of boolean comparison in your code.

```
// this is true, because JS was told to convert things until both left and right are boolean:
"" == 0 

// this is also true, for the same reason
"cake" == 1

// this is false, because "" is string and 0 is number and these can't possibly be the same 
"" === 0 
```